contract/src/offer-up-proposal.js

```diff
@@ -3,7 +3,7 @@ import { E } from '@endo/far';
 import { makeMarshal } from '@endo/marshal';
```

Comment:
Nit: 
```
offer-up-proposal.js
offer-up.contract.js
```

Naming here seems inconsistent. Is there a convention we want to use?

Reply:
@turadg suggested `xyz.contract.js` at some point when we were talking about using almost-but-not-quite jessie rules for linting contracts. The idea is growing on me... I find myself wishing I could see which are the bundle roots at a glance now and then; so since I had to rename `gameAssetContract.js` anyway, I thought I'd go for it.

(_neaby: @turadg also pointed out `test-foo.js` in our [unit testing norms](https://github.com/Agoric/agoric-sdk/wiki/agoric-sdk-unit-testing) is counter to ava norms, which call for `foo.test.js`_)

in swaparoo, @dtribble did some renaming so that you could just set the contract name in one place; in particular, instead of [build-game1-start.js](https://github.com/Agoric/dapp-offer-up/blob/main/contract/scripts/build-game1-start.js) , he used [build-contract-deployer.js](https://github.com/agoric-labs/dapp-swaparoo/blob/main/contract/scripts/build-contract-deployer.js) . The name "deploy" is growing on me too. After all: the proposal is what goes out to the BLD stakers; the code here is what executes if the proposal carries. I'm less convinced about filenames that are contract-agnostic. I have multiple contracts in a package more often than not.

So if I'm doing any more renaming in this PR (and given the amount of manual testing, I am not excited about it) it will probably be to change `offer-up-proposal.js` to `offer-up.deploy.js`. But then... `startOfferUpContract` is the main function in there, which argues for `offer-up.start.js`... or something.